api/src/main/resources/messages.properties

```diff
@@ -258,6 +258,12 @@ adminui.visits.title=Visits
 adminui.manageVisitAttributeTypes.title=Manage Visit Attribute Types
 adminui.addNewVisitAttributeType.title=Add New Visit Attribute Type
 adminui.editVisitAttributeType.title=Edit Visit Attribute Type
+adminui.manageVisitTypes.title=Manage Visit Types
+adminui.addNewVisitType.title=Add New Visit Type
+adminui.editVisitType.title=Edit Visit Type
+adminui.visitType.retire.title=Retire Visit Type
```

Comment:
I think it's better to have the label for retire to be `Delete` while that for delete to be something _Delete Forever_ or _Delete Permanently_, the reason for this is that from a regular user's perspective, they don't know that when they delete an item that we actually just retire/void it rather than purge it in the system, It's the tech survey system admin that might know this since it an internal implementation detail.

Reply:
I guess it's fine since that's how others have been implemted